model/project_invitation.rb

```diff
@@ -4,6 +4,10 @@
 
 class ProjectInvitation < Sequel::Model
   many_to_one :project
+
+  def expired_at
```

Comment:
nit: Maybe rename this as `expires_at`, because for most active records expiration date will be in the future (thus it is not expire**d** yet.

Reply:
Makes sense